www-apps/qtchan/patches/qtchan-0.95.patchset

```diff
@@ -0,0 +1,60 @@
+From 19fb61ef810b8356cd77f5a5720b439020a0fc0e Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Tue, 1 May 2018 16:36:05 +0200
+Subject: Do not create dot folders in $HOME
+
+
+diff --git a/mainwindow.cpp b/mainwindow.cpp
+index 8990d2e..16beea8 100644
+--- a/mainwindow.cpp
++++ b/mainwindow.cpp
+@@ -63,7 +63,11 @@ MainWindow::MainWindow(QWidget *parent) :
+ 	connect(&settingsView,&Settings::update,[=](QString field, QVariant value){
+ 		if(field == "use4chanPass" && value.toBool() == true){
+ 			QSettings settings(QSettings::IniFormat,QSettings::UserScope,"qtchan","qtchan");
+-			QString defaultCookies = QDir::homePath() + "/.config/qtchan/cookies";
++			#ifndef __HAIKU__
++				QString defaultCookies = QDir::homePath() + "/.config/qtchan/cookies";
++			#else
++				QString defaultCookies = QDir::homePath() + "/config/settings/qtchan/cookies";
++			#endif
```

Comment:
Since QtChan is an Qt5 program, you could change the patch to make use Qt's [`QStandardPaths::AppConfigLocation`](https://doc.qt.io/qt-5/qstandardpaths.html), which would returns the location of the configuration folder (it uses `find_directory()` internally for Haiku), with an upside of being easily upstreamable (it's cross-platform, and IMO that's how it should be done).